src/utils/getMenusFromDocs.ts

```diff
@@ -42,6 +42,8 @@ const getMenusFromDocs = (docs: Entry[]): Menus => {
       } else {
         uniqueMenus[menu].items.push(doc)
       }
+    } else if (!doc.routable) {
```

Comment:
This is a custom flag that we can add to the top of MDX files to get them to be ignored by the menu but still exist in the build for importing into other files